drivers/timer/ite_it8xxx2_timer.c

```diff
@@ -373,6 +414,38 @@ static int sys_clock_driver_init(const struct device *dev)
 		return ret;
 	}
 
+	if (IS_ENABLED(CONFIG_ARCH_HAS_CUSTOM_BUSY_WAIT)) {
+		/* Set timer5 and timer6 combinational mode for busy wait */
+		IT8XXX2_EXT_CTRLX(BUSY_WAIT_L_TIMER) |= IT8XXX2_EXT_ETXCOMB;
+
+		/* Set 32-bit timer6 count down 1 unit in every 1us */
+		ret = timer_init(BUSY_WAIT_H_TIMER, EXT_PSR_8M, EXT_RAW_CNT,
+				 BUSY_WAIT_TIMER_H_MAX_CNT, EXT_FIRST_TIME_ENABLE,
+				 BUSY_WAIT_H_TIMER_IRQ, BUSY_WAIT_H_TIMER_FLAG,
+				 EXT_WITHOUT_TIMER_INT, EXT_START_TIMER);
+		if (ret < 0) {
+			LOG_ERR("Init busy wait high timer failed");
+			return ret;
+		}
+
+		/*
+		 * Set 24-bit timer5 overflow in every 1us
+		 * NOTE: When everytime the timer5 count down from 0x7 to overflow,
```

Comment:
`Every time time5 counts down from [...]`?

Reply:
I re-write the comment, I think it's better to read now.